go/cmd/api-gateway/consume.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Jorrit05/DYNAMOS/pkg/lib"
+	pb "github.com/Jorrit05/DYNAMOS/pkg/proto"
+)
+
+func handleIncomingMessages(ctx context.Context, grpcMsg *pb.SideCarMessage) error {
```

Comment:
General comment: we should probably add short 'functional' comment lines to explain a bit more of why a function is there and doing something/

(Although I should've done that more in my thesis :) )